lib/state/appState.js

```diff
@@ -89,6 +89,17 @@ appState.getInitial = function() {
   };
 };
 
+appState.hideUnavailableDevices = function() {
+  var currentOS = this.app.device._os; // "mac", "win", "android", "cros", "linux", or "openbsd"
```

Comment:
On second thought, I see we're already using `typeof chrome !== 'undefined'` in `appActions.js` to ignore chrome-specific code in the test environment. I'm going to add a `getOS()` function to `appActions.js`, which will create `this.app._os` on load.